ports/rp2/boards/ARDUINO_NANO_RP2040_CONNECT/mpconfigboard.h

```diff
@@ -0,0 +1,39 @@
+//Board config for Arduino Nano RP2040 Connect.
+
+// Board and hardware specific configuration
+#define MICROPY_HW_BOARD_NAME           "Arduino Nano RP2040 Connect"
+#define MICROPY_HW_FLASH_STORAGE_BYTES  (8 * 1024 * 1024)
+
+// Enable networking and sockets.
+#define MICROPY_PY_NETWORK              (1)
+#define MICROPY_PY_USOCKET              (1)
+
+// Enable USB Mass Storage with FatFS filesystem.
+#define MICROPY_HW_USB_MSC              (1)
+#define MICROPY_HW_USB_VID              (0x2341)
+#define MICROPY_HW_USB_PID              (0x015e)
```

Comment:
Are these the Arduino VID/PID?  I think they should be the MicroPython ones used in stm32, because with this firmware installed it's not running Arduino anymore... not sure exactly what the correct thing is here.

Reply:
Yes these are indeed Arduino's VID/PID assigned to this board when running MicroPython, actually when it's running our firmware, but it's based on MicroPython so it will do for now and we may change the PID later. I saw a discussion about that before, and I think the VID/PID should uniquely identify the hardware and software not just the software which can be running on different boards. Knowing that some board is running MicroPython is not very useful on its own, if we can identify the board we can for example update a bootloader or revert its firmware etc.. just my view on the issue.

Reply:
Ok, let's leave them as-is (the PICO actually has custom VID/PID for PICO running MicroPython).